prow/jobs/images/build-docs.sh

```diff
@@ -27,7 +27,7 @@ echo "build-docs.sh] 📝 Installing requirements file... "
 pip install -r requirements.txt
 
 echo -n "build-docs.sh] 📄 Generating services page... "
-python3 ./scripts/gen_services.py
+python3 ./scripts/gen_services.py --debug
```

Comment:
did you intend to leave in the debug flag?

Reply:
Yep, next time we'll be able to debug these issues via prow logs directly..